openshift-knative-operator/pkg/serving/secretinformerfiltering.go

```diff
@@ -0,0 +1,81 @@
+package serving
+
+import (
+	"strconv"
+
+	mf "github.com/manifestival/manifestival"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"knative.dev/networking/pkg/apis/networking"
+	"knative.dev/operator/pkg/apis/operator/base"
+	operatorv1beta1 "knative.dev/operator/pkg/apis/operator/v1beta1"
+)
+
+const (
+	// TODO: remove when available in knative.dev/networking/config
+	ServingInternalCertName = "knative-serving-certs"
+	// TODO: Maybe decide to fetch from net-kourier deps instead
+	EnableSecretInformerFilteringByCertUIDEnv = "ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID"
+)
+
+func enableSecretInformerFiltering(ks base.KComponent) mf.Transformer {
+	shouldInject := false
+	for _, dep := range []string{"net-istio-controller", "net-kourier-controller"} {
+		if configIfUnsetAndCheckIfShouldInject(ks, dep, "controller") {
+			shouldInject = true
+		}
+	}
+	if shouldInject {
+		return injectLabelIntoInternalEncryptionSecret()
+	}
+	return nil
+}
+
+func injectLabelIntoInternalEncryptionSecret() mf.Transformer {
+	return func(u *unstructured.Unstructured) error {
+		if u.GetKind() == "Secret" && u.GetName() == ServingInternalCertName {
+			labels := u.GetLabels()
+			if labels == nil {
+				labels = make(map[string]string, 1)
+			}
+			labels[networking.CertificateUIDLabelKey] = "data-plane"
+			u.SetLabels(labels)
+			return nil
+		}
+		return nil
+	}
+}
+
+// Adds default (true) to env vars for secret informer filtering in net-* deployments and returns if we should inject
```

Comment:
Sorry if I misunderstood but we will add the default to `true` on v1.26?
I thought we will move it to 1.27 as per your comment in https://issues.redhat.com/browse/SRVKS-944

Reply:
I think docs team still has some time. Let me pinged them again. Doc issue is blocked on eng at the moment.